server/src/test/java/io/druid/initialization/InitializationTest.java

```diff
@@ -454,8 +454,8 @@ public void testExtensionsWithSameDirName() throws Exception
     final ClassLoader classLoader1 = Initialization.getClassLoaderForExtension(extension1, false);
     final ClassLoader classLoader2 = Initialization.getClassLoaderForExtension(extension2, false);
 
-    Assert.assertArrayEquals(new URL[]{jar1.toURL()}, ((URLClassLoader) classLoader1).getURLs());
-    Assert.assertArrayEquals(new URL[]{jar2.toURL()}, ((URLClassLoader) classLoader2).getURLs());
+    Assert.assertArrayEquals(new URL[]{jar1.toURI().toURL()}, ((URLClassLoader) classLoader1).getURLs());
```

Comment:
Why this?

Reply:
Because, `File.toURL()` is deprecated.

Reply:
Please add this method to our forbidden-apis file: https://github.com/apache/incubator-druid/blob/master/codestyle/druid-forbidden-apis.txt. When adding, please check that it works by intentionally leaving one call to this method in the code, and see if `mvn clean install -DskipTests` fails locally. For syntax, see https://github.com/policeman-tools/forbidden-apis/wiki/SignaturesSyntax.